README.md

```diff
@@ -14,6 +14,7 @@ Home Assistant Operating System uses Docker as its container engine. By default
 
 ## Supported hardware
 
+- Nabu Casa (official hardware from Home Assistant founders)
```

Comment:
```suggestion
- Nabu Casa
```

Other vendors/devices are listed without any comments, let's do it for NC as well (and keep it on the first place :innocent:).